Ngsa.App.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>net5.0</TargetFramework>
     <RootNamespace>Ngsa.Application</RootNamespace>
-    <AssemblyName>ngsa-aspnetapp</AssemblyName>
+    <AssemblyName>aspnetapp</AssemblyName>
```

Comment:
I think I remember for the customer deployment it needs to be aspnetapp in order to pick up metrics.  If that is true then leave it this way.  If not, let's use something more descriptive.